libs/crypto/ed25519/keypair.c

```diff
@@ -1,12 +1,12 @@
 #include "ed25519.h"
-#include "sha3.h"
+#include "sha512.h"
 #include "ge.h"
 
 
 void ed25519_create_keypair(unsigned char *public_key, unsigned char *private_key, const unsigned char *seed) {
     ge_p3 A;
 
-    sha3_512(seed, 32, private_key);
+    sha512(seed, 32, private_key);
```

Comment:
does size of private_key 64 bytes?

Reply:
Yes. The number here refers to seed length, and seed is 32 bytes.